mudirac/mudirac.xml

```diff
@@ -185,9 +205,24 @@ spec_expdec: $spec_expdec
             <option value="SPHERE"> finite size, uniformly charged spherical nucleus </option>
             <option value="FERMI2"> Fermi 2-term charge distribution </option>
         </param>
-        <param name="uehling_correction" type="boolean" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not" optional="true"/>
+        <conditional name="uehling_correction">
+            <param name="uehling_correction" type="select" display="radio" value="false" label="Uehling correction" help="whether to turn on the Uehling correction or not">
+                <option value="false" selected="true">False</option>
+                <option value="true">True</option>
+            </param>
+            <when value="false"/>
+            <when value="true">
+                <param name="uehling_steps" type="integer" value="" label="integration steps for the Uehling potential." help=" Higher numbers will make the Uehling energy more precise but increase computation times. Default is 100." optional="true"/>
+                <param name="uehling_lowcut" type="float" value="" label="low cutoff for Uehling potential" help="under low cutoff, the radius will be considered 0. Default is 0." optional="true"/>
+                <param name="uehling_highcut" type="float" value="" label="high cutoff for Uehling potential" help="over high cutoff, the radius will be considered r > > c . Default is INFINITY." optional="true"/>
```

Comment:
```suggestion
                <param name="uehling_highcut" type="float" value="" min="0" label="high cutoff for Uehling potential" help="over high cutoff, the radius will be considered r > > c . Default is INFINITY." optional="true"/>
```

As above, this should not be negative (but should still allow it to be unset to get the default infinite behaviour). To be honest I kind of think we should also check that it's greater than the lowcut, as that doesn't make sense to me either, but think that's a change for mudirac rather than the wrapper.